control_msgs/msg/JointTrajectoryControllerState.msg

```diff
@@ -1,11 +1,30 @@
+# This message presents current controller state of JTC
+
+# Header timestamp should be update time of controller state
 std_msgs/Header header
 
 string[] joint_names
+# The set point, that is, desired state.
+trajectory_msgs/JointTrajectoryPoint reference
+# Current value of the process (ie: latest sensor measurement on the controlled value).
+trajectory_msgs/JointTrajectoryPoint feedback
+# The error of the controlled value, essentially reference - feedback (for a regular PID implementation).
+trajectory_msgs/JointTrajectoryPoint error
+# Current output of the controller.
+trajectory_msgs/JointTrajectoryPoint output
+# -- deprecated --
 trajectory_msgs/JointTrajectoryPoint desired
 trajectory_msgs/JointTrajectoryPoint actual
-trajectory_msgs/JointTrajectoryPoint error  # Redundant, but useful
```

Comment:
There are a few removed parts, I think we shouldnt remove em

Reply:
you mean `error` and `multi_dof_error`? they just were moved up to have all deprecated ones at the end.

Reply:
Ah my bad, I couldn't find them